### PR TITLE
B2 URI cmds

### PR DESCRIPTION
**Pull request by mjurbanski-reef**

### CODE REVIEW

File: b2/console_tool.py [hunk 1]
```diff
@@ -509,6 +506,37 @@ def _get_file_name_from_args(self, args):
...
+    @classmethod
+    def _setup_parser(cls, parser):
+        add_b2_file_argument(parser)
+        super()._setup_parser(parser)
+
+    def get_b2_uri_from_arg(self, args: argparse.Namespace) -> B2URIBase:
+        return args.B2_URI
+
+
+class B2URIFileIDArgMixin:
```
Comment (jsuchan-reef) [inline b2/console_tool.py:519]: nit: while I generally prefer to use uppercased `ID`, I have mixed feelings about it here as it operates with non-uppercase names (`fileId` `B2FileIdURI`)

**Review by jsuchan-reef — APPROVED** [review]

File: b2/console_tool.py (hunk 1 above)
Comment (mjurbanski-reef) [reply b2/console_tool.py:519]: this is used visible, which is only why i created it upercase

i.e. it shows up in `--help`, and I wanted it to be consistent with documentation "B2 URI"

File: b2/_cli/b2args.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    return b2_uri
+
+
+B2_URI_ARG_TYPE = wrap_with_argument_type_error(parse_b2_uri)
+B2_URI_FILE_ARG_TYPE = wrap_with_argument_type_error(b2_file_uri)
+
+
+def add_b2_file_argument(parser: argparse.ArgumentParser, name="B2_URI"):
+    """
+    Add an argument to the parser that must be a B2 URI pointing to a file.
```
Comment (jsuchan-reef) [inline b2/_cli/b2args.py:38]: nit: some docstrings are in imperative mood, some of them are in descriptive mood.

Comment (mjurbanski-reef) [reply b2/_cli/b2args.py:38]: imperative mood is recommended by 257 for docstrings, but now I look at this it seems for Commands we are using descriptive mood. Not sure if I should change it as part of this PR.

**Pull request merged**